audbcards/core/dataset.py

```diff
@@ -25,12 +43,87 @@ class Dataset:
         cache_root: cache folder
 
     """
+
+    def __new__(
+        cls,
+        name: str,
+        version: str,
+        *,
+        cache_root: str = "~/.cache",
+    ):
+        r"""Create Dataset Instance."""
+        instance = _Dataset.create(name, version, cache_root=cache_root)
+
+        return instance
+
+    @staticmethod
+    def _map_iso_languages(*args):
+        return _Dataset._map_iso_languages(*args)
+
+    @staticmethod
+    def _dataset_cache_path(*args):
+        cache_path = _Dataset._dataset_cache_path(*args)
+        return cache_path
+
+    @staticmethod
+    def _load_pickled(path: str):
+        ds = _Dataset._load_pickled(path)
+        return ds
+
+    @staticmethod
+    def _save_pickled(obj, path: str):
+        """Save object instance to path as pickle."""
+        return _Dataset._save_pickled(obj, path)
+
+
+class _Dataset:
+    @classmethod
+    def create(
+        cls,
+        name: str,
+        version: str,
+        *,
+        cache_root: str = "~/.cache/",
```

Comment:
Again, please specify the actual path of the cache folder here.

Reply:
set to `~/.cache/audbcards`